pages/service/[id].tsx

```diff
@@ -208,13 +258,63 @@ export const ServicePage = ({ serviceData }: ServicePageProps): JSX.Element => {
         ) : null}
       </Section>
 
+      {/* Access info */}
       {serviceData.access ? (
         <Section divider={MyBestLifeTheme.colours.aqua}>
           <Heading as="h2">How do I access it?</Heading>
           <p>{serviceData.access}</p>
         </Section>
       ) : null}
 
+      {/* Contact information */}
+      <Section divider={'transparent'}>
+        {serviceData.contactExplanation ? (
+          <>
+            <Heading as="h2">How to get in touch</Heading>
+            <p>{serviceData.contactExplanation}</p>
+          </>
+        ) : null}
+
+        <Heading as="h3">Contact Details</Heading>
+        <ContactList>
+          {serviceData.email ? (
+            <>
+              <EmailListItem>
+                <span>Email: </span>
+                {serviceData.email}
+              </EmailListItem>
+            </>
+          ) : null}
+          {serviceData.form ? (
+            <>
+              <ContactListItem>
+                <ContactLink href={`${serviceData.form}`}>
+                  <span>Form</span>
+                </ContactLink>
+              </ContactListItem>
+            </>
+          ) : null}
+          {serviceData.phone ? (
+            <>
+              <ContactListItem>
+                <ContactLink href={`tel:${serviceData.phone}`}>
+                  <span>Phone</span>
+                </ContactLink>
+              </ContactListItem>
+            </>
+          ) : null}
+          {serviceData.website ? (
+            <>
+              <ContactListItem>
+                <ContactLink href={`${serviceData.website}`}>
+                  <span>Website</span>
+                </ContactLink>
+              </ContactListItem>
+            </>
+          ) : null}
```

Comment:
1. (minor) For each of these sections the fragment is redundant.

2. (improvement) It would be nice the the `ContactLink` component was responsible for creating the inner `span` element. Currently the inner span is referenced by the `EmailListItem` whereas it would be nicer if it was self contained within by the `ContactLink` component. And (in this case minor) nicer still if the CSS selector was scoped so that instead of `span` it was e.g. `${ContactLinkText} { .. }` :slightly_smiling_face:

Reply:
These are both true, thank you for your attention to detail! I have added to the list here: https://trello.com/c/Y77bYSpX/54-testing-issues